commands/show.go

```
@@ -141,11 +153,20 @@ func runShow(env *Env, opts showOptions, args []string) error {
 				env.out.Printf("%s\n", p.DisplayName())
 			}
 		case "ccb":
-			env.out.Printf("%s\n", strings.Join(ccbSummary(snap), "\n"))
+			if ccbState := ccbSummary(snap); ccbState != nil {
+				env.out.Printf("%s\n", strings.Join(ccbState, "\n"))
+			}
 		case "shortId":
 			env.out.Printf("%s\n", snap.Id().Human())
 		case "status":
 			env.out.Printf("%s\n", snap.Status)
+		case "nextStatuses":
+			validStatuses, err := snap.NextStatuses()
+			if err == nil && validStatuses != nil {
```

add a comment why non-nil err is ignored